Index: calamares-settings-blankon/scripts/bootloader-config
===================================================================
--- calamares-settings-blankon.orig/scripts/bootloader-config
+++ calamares-settings-blankon/scripts/bootloader-config
@@ -14,7 +14,7 @@ echo "Running bootloader-config..."
 
 if [ -d /sys/firmware/efi/efivars ]; then
     echo " * Installing grub-efi (uefi)..."
-    DEBIAN_FRONTEND=noninteractive chroot $CHROOT apt-get -y install grub-efi-amd64 cryptsetup keyutils
+    DEBIAN_FRONTEND=noninteractive chroot $CHROOT apt-get -y install --no-install-recommends efibootmgr grub-efi-amd64 cryptsetup keyutils
 else
     echo " * install grub... (bios)"
     DEBIAN_FRONTEND=noninteractive chroot $CHROOT apt-get -y install grub-pc cryptsetup keyutils
